Sources/FronteggSwift/embedded/EmbeddedLoginPage.swift

```diff
@@ -11,16 +11,19 @@ import UIKit
 
 public struct EmbeddedLoginPage: View {
     @EnvironmentObject var fronteggAuth: FronteggAuth
+    private let loginHint: String?
     
-    public init() {}
+    public init(loginHint: String? = nil) {
+        self.loginHint = loginHint
+    }
     
     
     public var body: some View {
         ZStack {
             NavigationView{
                 VStack(alignment: .center) {
                     ZStack{
-                        FronteggWebView()
+                        FronteggWebView(loginHint: loginHint)
```

Comment:
Do not add any constuctor parameter to frontegg views

Reply:
Done